src/main/java/vg/civcraft/mc/citadel/ReinforcementLogic.java

```diff
@@ -210,15 +210,11 @@ public static Reinforcement resolveDoubleChestReinforcement(Block block) {
 		if (rein != null || (mat != Material.CHEST && mat != Material.TRAPPED_CHEST)) {
 			return rein;
 		}
-		for (Block relative : BlockAPI.getPlanarSides(block)) {
-			if (relative.getType() != mat) {
-				continue;
-			}
-			rein = getReinforcementAt(relative.getLocation());
-			if (rein != null) {
-				return rein;
-			}
+
+		Block otherHalf = BlockAPI.getOtherDoubleChestBlock(block);
+		if (otherHalf == null) {
+			return null;
 		}
-		return null;
+		return rein = getReinforcementAt(otherHalf.getLocation());
```

Comment:
Just return the reinforcement directly, you don't need to assign it to a variable (which will be discarded anyway when the method ends).